src/numa_threadpool.rs

```diff
@@ -182,7 +182,7 @@ mod numa {
             if unsafe { numa_bitmask_isbitset(numa_all_nodes_ptr, i) } {
                 let mut info = NodeInfo { node_id: (i as c_uint), cpuset: Vec::new(), context: context_ctor(i) };
 
-                let mut bitmask = unsafe { numa_allocate_cpumask() };
+                let  bitmask = unsafe { numa_allocate_cpumask() };
```

Comment:
What does this do?

Reply:
This is fixing another warning that bitmask didn't need to be mutable.